implement adding project to favorites issue p.7.6

Andrey1079 opened the pull request:
Создан компонент ProjectLike Button.
На странице Projects добавлен компонент лайка. Обертка CardProject заменена с Link на div, добавлен обработчик клика handleProject, удален вызов функции дубль getNextPrev, заменен на getAllProjects.
В компоненте ProjectView добавлен компонент лайка.
В api/projects созданы 2 функции добавление и удаление из избранного
В api/organizer удален getNextPrev, доработан функционал getAllProjects и getProjectById, для получения массива проектов и проекта зарегистрированным и не зарегистрированным пользователем.

Commit:
feat(projectview): add realise like and dislike project faeture  p7.6

fix #160

Commit:
feat(cardpriject projects projectview): implement adding a project to favorites

implement adding a project to favorites from projects page and projectView page and adding condition
isLoggedIn for display likebtn

#160

DBKai commented on src/utils/api/endpoints.js:
```diff
@@ -25,3 +25,5 @@ export const ENDPOINT_RESEND_ACTIVATION_USER = 'auth/resend_activation/';
 export const ENDPOINT_ORGANIZER_PROJECT_CATEGORIES = 'project_categories/';
 export const ENDPOINT_ORGANIZER_PROJECTS = 'projects/';
+
+export const ENDPOINT_PROJECTS = 'projects/';
```
Есть же endpoint ENDPOINT_ORGANIZER_PROJECTS, можно использовать его

Andrey1079 replied:
не заметил что эндпоинт уже существует :-)

Commit:
Merge branch 'develop' into feature/p.7.6_add_project_to_favorite

Commit:
fix(projects projectview organaizer, projectlikebutton): implement adding project for favorites

change logic of getting projects  array for logged and unlogged user, change size of likebtn for
diff pages

Andrey1079 committed:
Merge branch 'develop' into feature/p.7.6_add_project_to_favorite
